cms/templates/settings_graders.html

```diff
@@ -25,7 +27,11 @@
 </%block>
 <%block name="requirejs">
     require(["js/factories/settings_graders"], function(SettingsGradersFactory) {
-        SettingsGradersFactory(_.extend(${escape_json_dumps(course_details, cls=CourseSettingsEncoder) | n}, {is_credit_course: ${escape_json_dumps(is_credit_course) | n}}), "${grading_url}");
+        SettingsGradersFactory(
+            _.extend(${escape_jsoon_for_js(course_details, cls=CourseSettingsEncoder) | n},
```

Comment:
Your general pattern is to put escape_&#*!_for_js in the filter list at the end. Let's be consistent, but with nice spacing. :)